tests/test_ProblemData.py

```diff
@@ -158,6 +158,30 @@ def test_problem_data_raises_when_no_depot_is_provided():
     )
 
 
+def test_problem_data_raises_when_no_vehicle_type_is_provided():
+    """
+    Tests that the ``ProblemData`` constructor raises a ``ValueError`` when
+    no vehicle types are provided.
+    """
+    with assert_raises(ValueError):
+        ProblemData(
+            clients=[],
+            depots=[Depot(x=0, y=0)],
+            vehicle_types=[],
+            distance_matrices=[np.asarray([[]], dtype=int)],
+            duration_matrices=[np.asarray([[]], dtype=int)],
```

Comment:
This also raises, no? (inconsistent size)

I think we should have the exact same dimensions here as in the non-raise case below.

Reply:
good catch 👍